pkg/framework/helm.go

```diff
@@ -5,11 +5,18 @@ import (
 	"os"
 	"os/exec"
 	"strings"
+
+	"github.com/giantswarm/microerror"
 )
 
 // HelmCmd executes a helm command.
 func HelmCmd(cmd string) error {
-	return runCmd("helm " + cmd)
+	err := runCmd("helm " + cmd)
```

Comment:
Can we please stop using this? It makes me very very sad.

Reply:
Yes. Sit tight.